tutorials/plot_epochs_to_data_frame.py

```diff
@@ -102,7 +102,7 @@ class provides many useful methods for restructuring, reshaping and visualizing
 raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
 event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
 
-raw = mne.io.read_raw_fif(raw_fname)
+raw = mne.io.read_raw_fif(raw_fname, add_eeg_ref=False)
```

Comment:
does it affect the plots not to do?

raw.set_eeg_reference()  # set EEG average reference